rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -277,12 +278,27 @@ impl RoadBuilder {
             },
         };
 
-        Ok(RoadBuilder {
+        let mut road = RoadBuilder {
             forward_lanes,
             backward_lanes,
             highway,
             oneway,
-        })
+        };
+
+        if tags.is("motorroad", "yes") {
+            if let Some(c) = &locale.country {
+                if c == &Country::the_netherlands() {
+                    for lane in road.lanes_ltr_mut(locale) {
+                        lane.width = Width {
+                            target: Infer::Default(Metre::new(3.35)),
+                            ..Default::default()
+                        }
+                    }
+                }
+            }
+        }
```

Comment:
an obvious hack... so during some refactors I will look for nicer ways to house this code.

Reply:
Better to start with something! I like how code is organized by part of the road now. Maybe locale-specific widths should act like a cross-cutting concern and be housed in their own module, and act as a post-processing step. Then we can check if we're in the Netherlands, and if so, walk through everything and add detail. But probably best to have at least one other locale-specific example before trying to refactor